Civi/Api4/Translation.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Civi\Api4;
+
+/**
+ * Attach supplemental translations to strings stored in the database.
+ *
+ * @package Civi\Api4
+ */
+class Translation extends Generic\DAOEntity {
+
+  public static function permissions() {
+    return [
+      'meta' => ['access CiviCRM'],
```

Comment:
Will this be too strict for some outward facing entities?

Reply:
I don't think so. My impression is that `access CiviCRM` was the ordinary value for `meta`.

For reference, the `MessageTemplate`, `Event`, and `ContributionPage` are all things which would plausibly be translated. In all three of those, the `meta` requires `access CiviCRM`.

Another analog might be `Note` (since those can be attached to various things). It too sets `meta` to `access CiviCRM`.

Reply:
Proceeding to the subsequent line after the comment (`default => translate CiviCRM`)... performing a translation requires satisfying two checks

1. One needs the coarse guardian permission (i.e. *Yes, generally speaking, you can work with translations*)
2. One needs specific access to the record which is being translated.  (That's the `CRM_Core_DynamicFKAccessTrait` line above.)

Hypothetically, suppose we enabled this style of translation for `Event`s and `MessageTemplate`s. This arrangement means that:

* Someone with `edit message templates` and `translate CiviCRM` may do translations for a message template. (But they may not be able to translate events.)
* Someone with `edit all events` and `translate CiviCRM` may do translations for events. (But they may not be able to translate message templates.)